nodejs/templates/deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- ( include "labels" . ) | indent 4 }}
 spec:
+  revisionHistoryLimit: 0
```

Comment:
Any thoughts on making it default to 0, but allow overriding ?

Reply:
nah - can't see us ever using that.  I don't want to add complexity where we don't use it